demos/palm/python/docs-agent/apps_script/exportmd.gs

```diff
@@ -0,0 +1 @@
+/usr/local/google/home/nickvander/docs-agent-github3/third_party/docs-agent/g2docsmd-html/exportmd.gs
```

Comment:
I think this needs to be a relative symlink so that it works for everyone.

Reply:
Thanks for catching that! Fixed